diagnostic.md

```diff
@@ -45,7 +45,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
```

Comment:
Incorrect. The loop will run three times, making the value 8. Always feel free to test questions like this in replit or node.